backend/src/main/java/de/aljoschanyang/capstoneprojectfiturae/models/WorkoutEditDTO.java

```diff
@@ -0,0 +1,14 @@
+package de.aljoschanyang.capstoneprojectfiturae.models;
+
+import lombok.Builder;
+
+import java.util.List;
+
+@Builder
+public record WorkoutEditDTO(
+        String workoutName,
```

Comment:
```suggestion
        String name,
```
naming convention -> nicht nötig noch mal workout vor der eigentlichen eigenschaft hinzuschreiben. Liest sich immer etwas "doof" "workout.workoutName".

Reply:
Achso ok. Wird angepasst